src/Dropdown.php

```diff
@@ -1106,7 +1106,7 @@ public static function getDeviceItemTypes()
      *
      * @return array (group of dropdown) of array (itemtype => localized name)
      **/
-    public static function getStandardDropdownItemTypes()
+    public static function getStandardDropdownItemTypes($withLocalizedNames = true)
```

Comment:
Why do you need to disable translations here ?
The PHP doc says it return an array of  `itemtype => localized name` (i did not check if it is really the case).

Removing translation mean you get an `itemtype => itemtype` array, no ?
In this case, calling array_keys on the original function should be enough.